components/script/dom/bindings/utils.rs

```diff
@@ -133,15 +132,6 @@ pub fn to_frozen_array<T: ToJSValConvertible>(convertibles: &[T], cx: SafeJSCont
     *ports
 }
 
-/// Creates a Float32 array
-pub fn create_typed_array(cx: SafeJSContext, src: &[f32], dst: &Heap<*mut JSObject>) {
```

Comment:
I needed to remove this in order to fix build warnings

Reply:
Good to remove it.